modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Adding the configuration interactively
```

Comment:
I'd make this heading more descriptive since it doesn't explain what the configuration is for, maybe something more like "Managing function secrets and config maps by using the Knative CLI"?